pKa_challenge_instructions.md

```diff
@@ -0,0 +1,115 @@
+
+This method is capable of measuring pKas between 2 and 12, when protonatable groups are at a maximum distance of 4-5 atoms away from chromophores, so that a change of protonation state affects the absorbance of UV-chromophore. We have selected compounds where titratable groups are close to potential chromophores (conjugated rings), but it is possible to completely miss detection of a pKa value if a titratable group is not in interaction with a UV-chromophore.  
+
+pKa measurements of soluble compounds were performed in ionic-strength adjusted water with 0.15 M KCl. Indications for precipitation was observed by visual inspection of samples and also by inspection of UV-spectra plots as a function of pH at 500-600 nm region of the spectra. For compounds with insufficient solubility, cosolvent protocol is used where 3 UV-metric pKa measurements were done at different cosolvent:water ratios (typically 30%, 40% and 50% methanol) and  Yasuda-Shedlovsky extrapolation method is used to estimate pKa value at 0% cosolvent.
+
+Three replicate pKa measurements were made for all compounds at room temperature (25°C). Multiwavelength absorbance analysis of Sirius T3 allows very good resolution pKas but essentially this method measures macroscopic pKas. Microscopic pKas with with very close pKa values and overlapping changes in absorbance spectra could be measured as one macroscopic pKa value.
+
+## Instructions and Submission Templates
+Three types of predictions will be accepted. Participants are encouraged to submit their results in all or multiple submission types as it fits to their prediction methods.
+
+#### Type I - microscopic pKas and related microstates
+Predicting microscopic pKa values and related microstates structures. Different protonation states and tautomers constitute different microstates. 
+- Fill the `typeI_microscopic_pKas_and_microstates.csv` template for all molecules.
+- For each molecule, report as many microscopic pKas based on your best guest.
+- Record the pair of microstates IDs of microstate structures pairs (protonated HA and deprotonated A) associated for each microscopic pKa. To determine the microstate ID for your predicted structure check the spreadsheets in [SAMPL6/physical_properties/pKa/microstates](SAMPL6/physical_properties/pKa/microstates).
+- If your predicted structure is not included in the list, contact us to [make a request for new microstate](mehtap.isik@choderalab.org). See more details are in missing microstates section.
+- Report microscopic pKa values with 2 decimals (e.g. 10.71).
+- Reporting standard error of the mean (SEM) is optional and should be reported with 2 decimals. (e.g. 1.02).
```

Comment:
Highly recommended? 

And, why two decimals? Doesn't this prescription encourage people to violate good error reporting practices? (i.e.. report your uncertainty to one significant digit and report the value to the same precision?)

Reply:
We have decided previously for pKa predictions to be reported with 2 decimal places (2.13, 4.05 etc.). I want to describe a way so that SEM reporting is also consistent ( for example 0.01, 0.10, or 1.00). Is this appropriate?
@davidlmobley I am not familiar with the rule of "reporting your uncertainty to one significant digit". What is your recommendation?

Reply:
If you really want to do two decimal places for some reason, that's fine -- this is not a major issue. Still, it's a bit odd -- imagine "3.47 +/- 3.02". What's the point of all the precision?

The rule I learned was "first round your uncertainty, then decide the precision of your value" -- so "3.47+/-3.02" becomes "3 +/- 3" (yeah), "3.47+/-0.302" becomes "3.5+/-0.3", and "3.47+/-0.0302" becomes "3.47+/-0.03", etc., so that your uncertainty always has one significant digit.

Reply:
Just for purposes of automated processing. In summarizing things for humans, we'll do the appropriate rounding in reporting.

Reply:
Fine by me, yes.